Gemfile

```diff
@@ -84,13 +84,14 @@ gem 'dm-zone-types',               '~> 0.3'
 gem 'timezone_local',              '~> 0.1.5'
 
 group(:development) do
-	gem "letter_opener"
+  gem "letter_opener"
   gem 'thin'
+  gem 'byebug'
```

Comment:
To fix "byebug requires Ruby version >= 2.0.0" issue, you simply need to update to "pry-byebug" by replacing this line:
```
  gem 'byebug'
with:
```
  gem 'pry-byebug', platform: [:ruby_20]